packages/bundler/src/runBundler.ts

```diff
@@ -1,15 +1,15 @@
 import ow from 'ow'
 import fs from 'fs'
 
-import { program } from 'commander'
-import { erc4337RuntimeVersion } from '@erc4337/common/dist/src'
+import { Command } from 'commander'
+import { erc4337RuntimeVersion } from '@erc4337/common'
 import { ethers, Wallet } from 'ethers'
 import { BaseProvider } from '@ethersproject/providers'
 
 import { BundlerConfig, bundlerConfigDefault, BundlerConfigShape } from './BundlerConfig'
 import { BundlerServer } from './BundlerServer'
 import { UserOpMethodHandler } from './UserOpMethodHandler'
-import { EntryPoint, EntryPoint__factory } from '@erc4337/common/dist/src/types'
+import { EntryPoint, EntryPoint__factory } from '@account-abstraction/contracts'
```

Comment:
Are you sure you want ethers.js types exported from the main package?

Reply:
Why not?
what else do you have to export from this typescript/javascript package?
I also export an "artifacts" folder, is someone is interested in the json files.